palo_alto_cortex_xdr/assets/logs/palo-alto-cortex-xdr.yaml

```diff
@@ -174,12 +174,12 @@ pipeline:
           preserveSource: false
           overrideOnConflict: false
         - type: attribute-remapper
-          name: Map `malicious_urls` to `http.url`
+          name: Map `malicious_urls` to `http.defanged.url`
           enabled: true
           sources:
             - malicious_urls
           sourceType: attribute
-          target: http.url
+          target: http.defanged.url
           targetType: attribute
           preserveSource: false
```

Comment:
I think you need to change preserveSource to true since we are updating the integration (cf [Attributes remapper should set correct preservceSource](https://datadoghq.atlassian.net/wiki/spaces/LB/pages/3622502423/Reviewing+remappers#Attribute-remappers-should-correctly-set-preserveSource)
I was trying your change on staging but its still populated the old field http.url without it

Reply:
@audesikorav Palo Alto Cortex XDR is not released yet, so it is currently not in use. Therefore, this change is part of new integration as a result we have set `preserveSource: false` .
Let us know if you still have a doubt.

Reply:
Reviewed the dashboards, can we add content to the dashboard screenshots?
The alert/incident overview  widget ("Severity over time") is empty, can we generate some information so the screenshot will have the widget show information in it?

Reply:
Images updated, Thanks.

Reply:
looks good, thanks. Approved.